installer/bundle/bundle_skel.sh

```diff
@@ -863,6 +844,12 @@ case "$installMode" in
             pkg_upd_list $DSC_PKG omsconfig $?
         fi
 
+        # Install SCX (and OMI)
+        [ -n "${forceFlag}" ] && FORCE="--force" || FORCE=""
+        [ -n "${debugMode}" ] && DEBUG="--debug" || DEBUG=""
+        ./bundles/${SCX_INSTALLER} --upgrade $FORCE $DEBUG
+        SCX_STATUS=$?
```

Comment:
Same here. In what cases does it make sense to continue installation if low-level dependencies couldn't be upgraded for some reason?

One might argue that it could improve the SLA, and I'd agree. But at what cost? Is it worth improving the SLA when you're running older versions of SCX that are no longer recommended?